src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.form {
+  margin: 20px auto;
+}
+
+.search-icon {
+  position: absolute;
+  height: 19px;
+  padding-left: 26px;
+  padding-top: 25px;
+}
+
+.search {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  color: #3d4e61;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+}
```

Comment:
In this class, you should leave common styles for both inputs, and move big search input related to style to another class

Reply:
If you need more context - feel free to ping me in your personal chat in the slack

Reply:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2-nd point